### DISCUSSION TITLE
월드 내 직접 제작 영상 사용

### DISCUSSION THREAD

**Discussion by diddntjd99** [naverz/zepeto-studio-kor#90] [5. Q&A]
안녕하세요,
저희 팀이 직접 만든 mp4 영상을 오프닝 영상으로 사용해도 괜찮나요?
공모전에 문제가 있을까요?

**Comment by beanyh** [comment]
아쉽게도 영상이 포함된 경우는 심사 승인 불가 요소입니다. 
https://studio.zepeto.me/guides/world-guidelines
<img width="820" alt="ZEPETO Studio 2021-12-16 14-31-33" src="https://user-images.githubusercontent.com/8156764/146313832-48654d7d-513a-4f78-900a-7343027c5bc2.png">

**Reply by mhkim0720** [reply]
혹시 영상이 포함된 패키지 파일은 업로드부터 실패 하나요? 400메가 정도 되는 패키지 파일 업로드 중인데 안 되서요.

**Reply by beanyh** [reply]
영상파일 때문에 업로드 부터 실패하지는 않습니다. 다른 이유로 실패하는 것 같습니다~
